util/parsec_project.py

```
@@ -17,58 +17,78 @@
     # but this needs to be kept in mind for complex projects.
     functions: dict[str, CFunction] = field(default_factory=dict)
 
-    # Enums, structs, and global_vars are all represented as lists of dictionaries.
     # For the exact format of these dictionaries, see the ParseC documentation (linked above).
     enums: list[dict[str, Any]] = field(default_factory=list)
     structs: list[dict[str, Any]] = field(default_factory=list)
     global_vars: list[dict[str, Any]] = field(default_factory=list)
 
-    def __init__(self, file_path: Path):
-        """Create an instance of ParsecFile from the file at `file_path`.
+    def __init__(self, input_path: Path):
```

Ruff (0.14.14)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)